Description: Fix interpreter path in 'ugreenchat.py'
 Packaged ugreenchat.py is placed to /usr/share/doc/uwsgi-common/examples, so
 Lintian warns about relative interpreter path with informational tag
 (I: example-interpreter-not-absolute).
Author: Leonid Borisenko <leo.borisenko@gmail.com>
Forwarded: not-needed
Last-Update: 2011-01-18

Index: uwsgi/ugreenchat.py
===================================================================
--- uwsgi-0.9.6.6.orig/ugreenchat.py	2011-01-17 07:29:02.000000000 +0200
+++ uwsgi-0.9.6.6/ugreenchat.py	2011-01-17 07:29:11.000000000 +0200
@@ -1,4 +1,4 @@
-#!./uwsgi
+#!/usr/bin/uwsgi
 import uwsgi
 
 from bottle import route, default_app, request, response, redirect
